notebooks/EVA-release-RSIDquery.ipynb

```diff
@@ -0,0 +1,179 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our general query provides an abundance of information including date the ID was created and if the variant has been remapped to a newer assembly. For this task we want to find the associated clustered SS IDs, species/assembly, & variant location"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can parse the JSON output to find data of interest and assign these to variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ss_list=[]\n",
+    "for record in output:\n",
+    "    ssroll = record['accession']\n",
+    "    ss_list.append(ssroll)\n",
+    "assembly = output[0]['data']['referenceSequenceAccession']\n",
+    "chrom = output[0]['data']['contig']\n",
+    "locat = output[0]['data']['start']\n",
+    "species = output[0]['data']['taxonomyAccession']\n",
+    "ref = output[0]['data']['referenceAllele']\n",
+    "alt = output[0]['data']['alternateAllele']\n",
+    "rsid = output[0]['data']['clusteredVariantAccession']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As the API uses INSDC nonmenclature, we can use another endpoint to convert the accessions to names"
```

Comment:
I think this comment belongs one cell down as the cell immediately below is just about getting scientific name & assembly name.

Also in this case getting the names isn't strictly necessary (it's just for making the output more readable), so I think it would be okay to skip that step, unless you think it fits the story.  Up to you.